src/uiTest/resources/project.rhino.js

```diff
@@ -298,16 +303,26 @@ function Project(underlyingProject) {
     let fullName = 'Git Machete: ' + name;
     const getButton = function () {
       // findAll() returns a LinkedHashSet
-      const result = robot.finder().findAll(component =>
-        component instanceof ActionButton && fullName.equals(component.getAction().getTemplatePresentation().getText())
-      ).toArray();
+      const result = robot.finder().findAll(component => {
+         const componentFound = component instanceof ActionButton
+           && fullName.equals(component.getAction().getTemplatePresentation().getText());
+         if (componentFound) {
+           if (component.isEnabled()) {
+             return true;
+           } else {
+            component.update();
+            component.updateUI();
```

Comment:
Whoops how the two are different? 😅 Also, `com.virtuslab.archunit.ForbiddenMethodsTestSuite#no_classes_should_call_JComponent_updateUI`... does it apply here? 🤔